src/content/developers/docs/transactions/index.md

```diff
@@ -305,6 +305,22 @@ Watch Austin walk you through transactions, gas, and mining.
 
 <iframe width="100%" height="315" src="https://www.youtube.com/embed/er-0ihqFQB0" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
+## Typed Transaction Envelope
+
+Ethereum has different transaction types that specify what a transaction should do. For example, sending Ether to an address, deploying a contract, etc.  
```

Comment:
This is a little inaccurate -- all tx types (so far) can send ether, deploy contracts, etc. These functions are considered different "types" of transactions. The different "types" have much more to do with allowing new transaction behavior that doesn't exist in the protocol already (e.g. access lists).

Reply:
Hey @lightclient, can you help me reframe or replace this statement?
Thanks!!!

Reply:
Maybe something like "Over time Ethereum has evolved to support multiple types of transactions. This allows new features like access lists and EIP-1559 to be implemented without affecting legacy transaction formats."?

Reply:
yeah that looks great!!
Thanks !